docs/resources/examples/wan_client.yaml

```diff
@@ -0,0 +1,108 @@
+######################
+# WAN CLIENT EXAMPLE #
+######################
+
+##################################
+# ALLOWED TOPICS
+# Allowing FastDDS and ROS2 HelloWorld demo examples topics
+
+allowlist:
+  [
+    {name: "HelloWorldTopic", type: "HelloWorld"},                # 1
+    {name: "rt/chatter", type: "std_msgs::msg::dds_::String_"},   # 2
+  ]
+
+##################################
+# SIMPLE PARTICIPANT
+# This participant will subscribe to topics in allowlist in domain 1 and listen every message published there
+
+SimpleParticipant:                                                # 3
+  type: local                                                     # 4
+  domain: 1                                                       # 5
+
+##################################
+# ROS DISCOVERY CLIENT
```

Comment:
Would this be a server?

Reply:
It will not, but I thought it would be too complex to explain that a Server without listening-address would be a SUPER_CLIENT, so I preferred to use id in all servers and clients

Reply:
But you are defining a listening-address here

Reply:
But you are defining a listening-address here

Reply:
True, cause in UDP both need listening-addresses.
It is specified in line 87